app/src/main/java/ca/marshallasch/veil/database/Database.java

```diff
@@ -1461,12 +1461,11 @@ public int getNumLost(@IntRange(from = 0, to = 3) int protocolVersion) {
      *
      * @param postHash the hash of the post item being marked
```

Comment:
Should also include in the comment that this will set the comments status as well to match the post status

Reply:
added java doc comment to include that comment hashes are also set